Equity-Analysis.ipynb

```diff
@@ -0,0 +1,675 @@
+    "[descriptors](https://techdocs.ed-fi.org/x/qQ-gBQ).\n",
+    "\n",
+    "Supported demographics†:\n",
+    "\n",
+    "* Disability\n",
+    "* Sex (note: existing data sets do not distinguish gender and sex, and only \n",
+    "  provide two options: male, female)\n",
+    "* Hispanic Ethnicity\n",
+    "* Language\n",
+    "* Limited English Proficiency\n",
+    "* Race\n",
+    "* Tribal Affiliation\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "† In the Ed-Fi Data Standard, these demographics are stored on the relationship\n",
+    "to an education organization - which can be with a school, an LEA, or other.\n",
+    "These sample data only use school or LEA relationships. <i>For more information on\n",
+    "this distinction between School and LEA demographics, please see <a \n",
+    "href=\"https://techdocs.ed-fi.org/x/CqwOB\">How to Use the Student Demographic \n",
+    "Dimensions</a></i>.\n",
+    "</div>\n",
+    "\n",
+    "Supported indicators:\n",
+    "\n",
+    "* **Attendance Rate**: based on \"negative attendance\" (assumed present unless marked\n",
+    "  as absent: `(Enrolled Days - Days Absent) / Enrolled Days`\n",
+    "* **Behavior**: number of disciplinary incidents reported during the school year\n",
+    "* **Course Performance**: grade average over all sections\n",
+    "\n",
+    "This notebook remains relatively course-grained, in that it does not attempt\n",
+    "to compare schools, grade levels, teachers, etc.\n",
+    "\n",
+    "## Acknowledgments and References\n",
+    "\n",
+    "Special thanks to Nancy Smith of [DataSmith Solutions,\n",
+    "LLC](http://datasmithsolutions.com/aboutus.html) for review and constructive\n",
+    "feedback on the first draft of this material, and to Shana Shaw\n",
+    "of the [Michael & Susan Dell Foundation](https://www.dell.org) for advice\n",
+    "on statistical inference.\n",
+    "\n",
+    "Ghasemi, Asghar, and Zahediasl, Saleh. [Normality Tests for Statistical Analysis: A Guide for\n",
+    "Non-Statisticians](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3693611/). _Int J Endocrinol\n",
+    "Metab_. 2012 Spring; 10(2): 486–489. Published online 2012 Apr 20. doi: 10.5812/ijem.3505.\n",
+    "\n",
+    "Cohen, Jacob. [Statistical Power Analysis for Behavior Sciences](https://www.google.com/books/edition/Statistical_Power_Analysis_for_the_Behav/2v9zDAsLvA0C). \n",
+    "United States, Taylor & Francis, 2013.\n",
+    "\n",
+    "Lock, Patti Frazer, et al. [Statistics: Unlocking the Power of Data](https://www.lock5stat.com/).\n",
+    "Third Edition. United States, Wiley, 2021.\n",
+    "\n",
+    "SciPy [API Reference](https://docs.scipy.org/doc/scipy/reference/index.html)\n",
+    "\n",
+    "\n",
+    "## Usage\n",
+    "\n",
+    "1. Requires Python 3.9 or 3.10 and [Poetry](https://python-poetry.org/).\n",
+    "1. You must have write access to a copy of an Ed-Fi ODS database, version 3.0 or newer.\n",
```

Comment:
Please include the instructions to install and use the poetry virtual environment